package.json

```diff
@@ -7,23 +7,17 @@
     "babel-loader": "^9.2.1",
     "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^7.1.2",
+    "gh-pages": "^6.3.0",
     "html-webpack-plugin": "^5.6.3",
     "jquery": "^3.7.1",
-    "mini-css-extract-plugin": "^2.9.2"
+    "mini-css-extract-plugin": "^2.9.2",
+    "save-dev": "^0.0.1-security"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack --mode production",
-    "dev": "webpack serve --mode development"
-  },
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "autoprefixer": "10.4.7",
-    "cssnano": "5.1.12",
-    "postcss-loader": "7.0.1",
-    "webpack": "^5.97.1",
-    "webpack-cli": "^6.0.1",
-    "webpack-dev-server": "^5.2.0"
```

Comment:
I can't start the project because some of the packages are missing from `package.json`